04-loop.md

```diff
@@ -277,7 +279,9 @@ Since she's still learning how to use the shell,
 she decides to build up the required commands in stages.
 Her first step is to make sure that she can select the right files --- remember,
 these are ones whose names end in 'A' or 'B', rather than 'Z'. Starting from her home directory, Nelle types:
-
+A graphical representation of how the loop will be processed is shown below:
+{insert image here: https://www.dropbox.com/s/vpn7k4r1gxiaaxo/Shell%20Script%20For%20Loop%20Flow%20Chart.jpg?dl=0]
+Figure: Flowcart of For Loop in Shell
```

Comment:
You are linking to the image already above (line 231). You can get rid of lines 282-284